app/views/layouts/_navigation_right.html.haml

```diff
@@ -2,6 +2,8 @@
   - if user_signed_in?
     - unread_count = current_user.notifications.unread.count
 
+    - unless Rails.env.test?
```

Comment:
switching things by environment here is bad practice, maybe use a flag in rad_common.yml instead